config/superglue-bow.conf

```diff
@@ -0,0 +1,86 @@
+// Config settings used for SuperGLUE CBoW baseline experiments. 
+
+// This imports the defaults, which can be overridden below.
+include "defaults.conf"
+exp_name = "superglue-bow"
+
+// Data and preprocessing settings
+max_seq_len = 256
+tokenizer = MosesTokenizer
+
+// Model settings
+sent_enc = "bow"
+word_embs = glove
+skip_embs = 0
+sep_embs_for_skip = 0
+elmo = 0
+classifier = log_reg // following BERT paper
+pair_attn = 0 // shouldn't be needed but JIC
+s2s = {
+    attention = none
+}
+transfer_paradigm = finetune
+
+// Training settings
+dropout = 0.1
+optimizer = adam
+batch_size = 16
+max_epochs = 10
+lr = .00001
+min_lr = .0000001
+lr_patience = 4
+patience = 20
+max_vals = 10000
+
+// Control flow
+do_pretrain = 1
+do_target_task_training = 1
+do_full_eval = 1
+write_preds = val,test
+write_strict_glue_format = 1
+
+// These override the task-specific configs set in defaults.conf
+cola = {}
+cola_classifier_dropout = 0.1
+cola_val_interval = 100
+cola_lr = 0.00001
+
+sts-b = {}
+sts-b_classifier_dropout = 0.1
+sts-b_val_interval = 1000
+sts-b_lr = 0.00001
+
+qnli = {}
+qnli_classifier_dropout = 0.1
+qnli_val_interval = 1000
+qnli_lr = 0.00001
+
+mnli = {}
+mnli_classifier_dropout = 0.1
+mnli_val_interval = 1000
+mnli_lr = 0.00001
+
+qqp = {}
+qqp_classifier_dropout = 0.1
+qqp_val_interval = 1000
+qqp_lr = 0.00001
+
+rte = {}
+rte_classifier_dropout = 0.1
+rte_val_interval = 100
+rte_lr = 0.00001
+
+wnli = {}
+wnli_classifier_dropout = 0.1
+wnli_val_interval = 100
+wnli_lr = 0.00001
+
+mrpc = {}
```

Comment:
Delete these non-SuperGLUE tasks?

Reply:
...and add the SuperGLUE ones?

Reply:
Will delete, but not sure what you mean about adding (there aren't any task-specific hyperparams).

Reply:
Got it. Assumed there were some that would be specified in the script, but makes sense.